### DISCUSSION TITLE
fix tabular editor 3 interface

### DISCUSSION THREAD

**Discussion by marceldcvvenado** [TabularEditor/TabularEditor3#554] [🙋♀️TE3 Q&A] [answered]
Hello World.
I was customizing the tabular editor 3 interface, trying to include buttons, unfortunately I started to reset the toolbar. I ended up losing all the toolbar icons including the menu bar. Seeing you ruin everything. I uninstalled tabular editor 3 and reinstalled it and it seems tabular editor saves all the customizations and when you reinstall it it stays the same. I need help please. I need the UI to go back to the default one.
Thank you
![interface error](https://user-images.githubusercontent.com/106497564/170897518-1c1148f6-213d-433e-a7e1-d9951cb37afe.PNG)

**Comment by otykier** [comment, marked as answer]
Delete the “Layouts.json” file in your `%localappdata%\TabularEditor3` folder.

**Reply by azhuang0528** [reply]
I also ran into this and this answer resolved it!  Thank you!!

**Comment by gabrielmission** [comment]
I have the similar problem except i don't see the "Layouts.json" file in my directory. Please help thanks!

**Reply by otykier** [reply]
Which files do you see in that directory?
